src/helm/proxy/clients/aleph_alpha_client.py

```diff
@@ -102,24 +123,21 @@ def do_it():
         )
 
     def tokenize(self, request: TokenizationRequest) -> TokenizationRequestResult:
-        """Make a request following https://docs.aleph-alpha.com/api/tokenize."""
-        raw_request = {
-            "model": request.tokenizer_name,
-            "prompt": request.text,
-            "tokens": True,
-            "token_ids": True,
-        }
-
+        """
+        Encode the text using Aleph Alpha's tokenizer library:
+        https://aleph-alpha-client.readthedocs.io/en/latest/aleph_alpha_client.html#aleph_alpha_client.Client.tokenizer
+        """
         try:
 
             def do_it():
-                result = self._send_request(AlephAlphaClient.TOKENIZE_ENDPOINT, raw_request)
-                assert "tokens" in result and "token_ids" in result, f"Invalid response: {result}"
-                return result
-
-            response, cached = self.cache.get(raw_request, wrap_request_time(do_it))
-        except (requests.exceptions.RequestException, AssertionError) as e:
-            error: str = f"AlephAlphaClient error: {e}"
+                tokenizer: Tokenizer = self._get_tokenizer(request.tokenizer_name)
+                result: Encoding = tokenizer.encode(request.text, add_special_tokens=False)
+                return {"token_ids": result.ids, "tokens": result.tokens}
+
+            cache_key = {"model": request.tokenizer_name, "prompt": request.text}
```

Comment:
Should we include `tokens` and `token_ids` in the cache key, to (1) maintain compatibility with the previous format, (2) document that this is a tokenization request rather than an inference request?